dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/TaskParametersUtils.java

```diff
@@ -88,6 +89,8 @@ public static AbstractParameters getParameters(String taskType, String parameter
                 return JSONUtils.parseObject(parameter, ConditionsParameters.class);
             case SQOOP:
                 return JSONUtils.parseObject(parameter, SqoopParameters.class);
+            case SWITCH:
+                return JSONUtils.parseObject(parameter, SwitchParameters.class);
```

Comment:
Please use new code of dev

Reply:
I fix the globalParam , then I will fix this .